VAGRANT.md

```diff
@@ -43,8 +43,8 @@ is.
       ```
       # inside the virtual machine:
       cd build
-      wget --no-verbose --output-document=/tmp/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
-      ./utils/setup.php --osm-file /tmp/monaco.osm.pbf --osm2pgsql-cache 1000 --all 2>&1 | tee monaco.$$.log
+      wget --no-verbose --output-document=/tmp/dump.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
+      cd /nominatim && nominatim import --osm-file /tmp/dump.osm.pbf
```

Comment:
The change to nominatim tool is fine in principle but this is not quite right.

The recommended way is along those lines:
```
cd nominatim-project
wget --no-verbose --output-document=monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
nominatim import --osm-file monaco.osm.pbf 2>&1 | tee monaco.$$.log
```

Reply:
Fixed